test/hopen/syntax/mustache_test.cljc

```diff
@@ -0,0 +1,101 @@
+(ns hopen.syntax.mustache-test
+  (:require [hopen.syntax.mustache :as sut]
+            #?(:clj [clojure.test :as t :refer [deftest testing is]]
+               :cljs [cljs.test :as t :include-macros true])))
+
+(deftest tag-parsing
+  (testing "Deliminator parsing"
+    (is (= (#'sut/parse-delim-change-tag "={{ }}=") '("{{" "}}")))
```

Comment:
Ah, you learned that trick to access private vars :-)
I originally learned it from @plexus 's booklet.

Reply:
Yeah, I was a bit surprised that bars can be used like that. I guess it makes sense. Vars need to be resolved to whatever it is before it can be used.